maze_transformer/evaluation/plot_maze.py

```diff
@@ -140,41 +161,72 @@ def show(self) -> None:
     def _rowcol_to_coord(self, points: CoordArray) -> NDArray:
         """Transform Points from MazeTransformer (row, column) notation to matplotlib default (x, y) notation where x is the horizontal axis."""
         points = np.array([(x, y) for (y, x) in points])
-        return self.UNIT_LENGTH * (points + 0.5)
+        return self.unit_length * (points + 0.5)
 
     def _plot_maze(self) -> None:
+        """
+        Define Colormap and plot maze.
+        Colormap: x < 0: black
+                  0 <= x <= self.max_node_value:
+                        fade from dark blue to white,
+                        upper bound adaptive to max node value
+        """
         img = self._latticemaze_to_img()
-        self.ax.imshow(img, cmap="gray", vmin=0, vmax=1)
-
-    def _latticemaze_to_img(
-        self, unit_length: int = UNIT_LENGTH
-    ) -> NDArray["row col", bool]:
+        if self.custom_node_value_flag is False:
+            self.ax.imshow(img, cmap="gray", vmin=-1, vmax=1)
+
+        else:  # if custom node_values have been passed
+            n_blues = int(256 * self.max_node_value)  # for scaling colorbar up to value
+            blues = mpl.colormaps["Blues"].resampled(n_blues)  # load blue spectrum
```

Comment:
Is there a reason for this to always be blue?

Reply:
Do we need an option for the user to plug other color spectra here? I assumed this is a choice we do once and it'll be a good fit for all experiments. `Blues` is good as we need a color spectrum from dark to light, which does go to black (to have the walls always visible).

Reply:
resolved
<img width="550" alt="Screenshot 2023-03-29 at 17 08 54" src="https://user-images.githubusercontent.com/61095597/228600395-95bfa1a0-9931-41e6-9b7f-61df2df0af8d.png">

Reply:
my reasoning from slack:

I imagine that we will want to display attention over the maze, and usually you use different colors for different heads. Being able to make the colors match would be useful for if we have a set of mazes with attention heatmaps for each and a set of actual attention matrices for each head.

Thanks for adding this!